source/prerequisites.txt

```diff
@@ -0,0 +1,24 @@
+.. _rm-prerequisites:
+
+=============
+Prerequisites
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The following pages outline the details and specifics of prepping your 
```

Comment:
I elected not to do the summary table here, we could possibly add as a fast follow once some of the major projects are landed for .local this year.